runtime/compiler/compile/J9SymbolReferenceTable.cpp

```diff
@@ -826,30 +826,8 @@ J9::SymbolReferenceTable::findOrFabricateShadowSymbol(
       qualifiedFieldName,
       TR::Symbol::UnknownField);
 
-   // As yet JITServer does not support getClassFromSignature() based directly
-   // on J9ConstantPool*, but it does support it using TR_OpaqueMethodBlock*.
-   // Find an arbitrary method (if there is one) defined by the same class that
-   // declares the field, and use that method to getClassFromSignature(), since
-   // it will have the desired constant pool.
-   //
-   // The class containing the field is highly likely to declare at least a
-   // constructor, and if it doesn't, then it seems that it's not possible to
-   // instantiate it in the usual way (new, dup, invokespecial <init>), so the
-   // performance of accesses to its instance fields is especially unlikely to
-   // matter.
-   //
    TR_J9VM *fej9 = reinterpret_cast<TR_J9VM *>(fe());
-   if (fej9->getNumMethods(containingClass) > 0)
-      {
-      auto *firstMethod =
-         static_cast<TR_OpaqueMethodBlock*>(fej9->getMethods(containingClass));
-
-      TR_OpaqueClassBlock *declaredClass = fej9->getClassFromSignature(
-         signature, (int32_t)strlen(signature), firstMethod);
-
-      if (declaredClass != NULL)
-         sym->setDeclaredClass(declaredClass);
-      }
+   TR_OpaqueClassBlock *declaredClass = fej9->getClassFromSignature(signature, (int32_t)strlen(signature), containingClass);
```

Comment:
@jdmpapin Is this something like what you wanted to write here?

Reply:
That's the `declaredClass` that I wanted to get, yes, but this change seems to be inadvertently deleting the call to `setDeclaredClass`

Reply:
Good catch. I think that got lost when I was editing a commit.